docs/fuzzing_binary-only_targets.md

```diff
@@ -113,10 +112,6 @@ The mode is approximately 2-5x slower than compile-time instrumentation, and is
 less conducive to parallelization. But for binary-only fuzzing, it gives a huge
 speed improvement if it is possible to use.
 
-If you want to fuzz a binary-only library, then you can fuzz it with frida-gum
-via frida_mode/. You will have to write a harness to call the target function in
-the library, use afl-frida.c as a template.
-
```

Comment:
Why is this removed?

Reply:
https://github.com/AFLplusplus/AFLplusplus/commit/0f299a3bef12b2a8f27a5ba629a2ecb1201fcd60